lib/components/Carleton360/Carleton360.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { PageHeader } from '../PageHeader/PageHeader'
+
+export interface Carleton360Props {
+  children?: React.ReactNode
+}
+
+export const Carleton360 = ({ children }: Carleton360Props) => {
+  const $componentMarginY = `my-6 md:my-12 first:mt-0`
+  return (
+    <div
+      className={`text-center relative flex items-center justify-center mx-auto px-8 overflow-hidden md:px-16 rounded-xl not-prose ${$componentMarginY} pt-10 pb-12  z-[2] max-w-4xl w-full  flex-col gap-2 bg-cu-black-50 cu-zero-first-last`}
+    >
+      <PageHeader header={''} as={'h2'} size={'md'} isWhite={false} isCenter noUnderline>
```

Comment:
Wait, I missed this before. We should not use page header here as we are not using a heading and you're passing an empty header which is an a11y issue.

This can be replaced with a wrapping div, not sure if you need any classes on it, doubtful.

I have updated the SVG itself, ran it through SVGOMG to clean it up. You can get the new SVG markup from that file and replace the image entirely.

On SVG you can add `className="mx-auto max-w-96"`